donate/templates/payment/card_upsell_master.html

```diff
@@ -41,6 +41,10 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Thank you
                             {% render_form_field form.amount %}
                         </div>
 
+                        {% for field in form.hidden_fields %}
```

Comment:
@alanmoo I don't actually know what this does... and whether it's needed or is the template equivalent of an accidental print()

Reply:
This spits out the currency in the upsell and I believe it's important to be able to grab the minimum for that currency.

Reply:
This injects the currency into the upsell view, which is necessary to be able to determine how much the minimum should be.